packages/backend/src/tests/tigerbeetle.ts

```diff
@@ -33,13 +33,16 @@ export async function startTigerbeetleContainer(clusterId?: number): Promise<{
       '--replica-count=1',
       `${TIGERBEETLE_DIR}/${tigerbeetleFile}`
     ])
+    .withPrivilegedMode()
```

Comment:
why don't we need to do this in the local playground?

Reply:
Good question... My local docker version is the same version that we suspect is causing a problem on github.  Not sure if there would be differences between docker verions on Mac/Ubuntu (or w/e the github environment is).